Packs/Phishing/Playbooks/Phishing_-_Generic_v3.yml

```diff
@@ -2409,10 +2216,10 @@ tasks:
       version: -1
       name: Predict Phishing URLs
       description: Predicts phishing URLs using a pre-trained model.
+      scriptName: DBotPredictURLPhishing
```

Comment:
This is probably because you worked on 6.5/7.0. Need to make sure that it is properly uploaded and works in 6.1 as well.